lib/active_model_serializers/adapter/json_api/pagination_links.rb

```diff
@@ -2,14 +2,21 @@ module ActiveModelSerializers
   module Adapter
     class JsonApi < Base
       class PaginationLinks
+        MissingSerializationContextError = Class.new(KeyError)
```

Comment:
Should this be defined within `PaginationLinks`? It seems like it's more general than that.